### PR TITLE
User activity stats from superadmin

### PR DESCRIPTION
**Pull request by alrocar**
Related to https://github.com/CartoDB/cartodb-central/issues/2455

### CODE REVIEW

File: app/models/user/user_decorator.rb [hunk 1]
```diff
@@ -2,6 +2,28 @@ module CartoDB
   module UserDecorator
     include AccountTypeHelper
 
+    def activity(options = {})
+      calls = options.fetch(:show_api_calls, true) ? get_api_calls(from: last_billing_cycle, to: Date.today) : []
+      map_views = calls.fill(0, calls.size..29).reduce(:+)
```
Comment (gonzaloriestra) [inline app/models/user/user_decorator.rb]: What is the point of adding 0s if you want to add all the values? :thinking:

Comment (alrocar) [reply app/models/user/user_decorator.rb]: mmm I copy/pasted this code and didn't think too much about it... it does not make sense

File: app/models/user/user_decorator.rb [hunk 2]
```diff
@@ -2,6 +2,28 @@ module CartoDB
   module UserDecorator
     include AccountTypeHelper
 
+    def activity(options = {})
+      calls = options.fetch(:show_api_calls, true) ? get_api_calls(from: last_billing_cycle, to: Date.today) : []
+      map_views = calls.reduce(0, :+)
```
Comment (gonzaloriestra) [inline app/models/user/user_decorator.rb]: You can just do `calls.reduce(:+)`, but it's ok.

**Comment by gonzaloriestra** [comment]
Acceptance made in https://github.com/CartoDB/cartodb-central/pull/2513

**Pull request merged**